terraform.tfvars.example

```diff
@@ -0,0 +1,11 @@
+# Example Terraform variables file
+# Copy this to terraform.tfvars and fill in your actual values
+# NEVER commit terraform.tfvars to version control
+
+project_id             = "genai-studio-mvp"
```

Comment:
Not sure if I'd even provide an example value for the project id...I'd force the user to set it.